net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Create BitTorrent files easily"
+HOMEPAGE="http://www.createtorrent.com/"
```

Comment:
I've tried to open this page few times on different days. It seems dead. Any alternatives?

Reply:
I think the page is dead, but there is another page with a somewhat newer file (which has the same version info) I need to check for possible use.

I could possibly fork the current code and support it directly if there is anyone else interested in this.  Do you know how many accesses the file has in the last year or so?